sw/device/silicon_creator/rom/BUILD

```diff
@@ -88,7 +88,7 @@ cc_library(
         "//hw/ip/gpio/data:gpio_regs",
         "//hw/ip/lc_ctrl/data:lc_ctrl_regs",
         "//hw/ip/otp_ctrl/data:otp_ctrl_regs",
-        "//hw/ip/pinmux/data:pinmux_regs",
+        "//hw/top_earlgrey/ip/pinmux/data/autogen:pinmux_regs",
```

Comment:
Oof, I might have done that. Could this be avoided in the future by prefixing/namespacing the symbols from one of those libraries? (Also, might need to run buildifier to sort these lines :)

Reply:
No worries. It looks like we already have #8440 for the long-term fix. But meanwhile we can consider not defining targets that we should not be using. Not sure if they will be easy to identify though. IIRC @tjaychen mentioned that most of those files should be the same but I'm not sure if that applies to peripherals like pinmux which are tightly coupled with a particular top.

Reply:
We could also set the target's visibility to an explicit allowlist.

Reply:
Most but not all :). Some top-level dependent ones may actually have different headers.

Reply:
At some point, we might want to consider defining an abstract "BSP-like" interface that the different top-level entities support, and bazel could select the correct implementation based on the target platform (where englishbreakfast and earlgrey would be different platforms, for example).